tools/ios-eddystone-scanner-sample/EddystoneScannerSample/ESSEddystone.m

```diff
@@ -167,23 +168,25 @@ + (instancetype)beaconInfoForUIDFrameData:(NSData *)UIDFrameData
 
   ESSEddystoneUIDFrameFields uidFrame;
 
-  if ([UIDFrameData length] != sizeof(ESSEddystoneUIDFrameFields)) {
-    return nil;
-  }
-  [UIDFrameData getBytes:&uidFrame length:sizeof(ESSEddystoneUIDFrameFields)];
+  if ([UIDFrameData length] == sizeof(ESSEddystoneUIDFrameFields) || [UIDFrameData length] == sizeof(ESSEddystoneUIDFrameFields) - 2) {
```

Comment:
Maybe best to replace the `-2` with `sizeof(uidframe.rfu)` as well?

Also, wrap at 100 chars, and indent is 4 (below). We are using [these style guidelines](http://google.github.io/styleguide/objcguide.xml) for our code.